plugins/generic/htmlArticleGalley/HtmlArticleGalleyPlugin.inc.php

```diff
@@ -100,6 +100,8 @@ function articleDownloadCallback($hookName, $args) {
 
 		if ($galley && $galley->getFileType() == 'text/html' && $galley->getFileId() == $fileId) {
 			echo $this->_getHtmlContents($request, $galley);
+			$returner = true;
+			HookRegistry::call('FileManager::downloadFileFinished', array(&$returner));
```

Comment:
Ah, yes, sure! Why hasn't this crossed my mind at all!? Although it seems so logical... :-\